spec.md

```diff
@@ -592,6 +594,204 @@ After implementing the first endpoint `GET /v2/catalog` documented
+
+\* Fields with an asterisk are REQUIRED.
+
+```
+{
+  "parameters": {
+    "database": {
+      "service_id": "90c7c453-c6a3-47ef-8284-5a2c00f4f18b",
+      "service_name": "the-database",
+      "plan_id": "d9301d66-7621-4526-9b97-63e93ddc2d13",
+      "plan_name": "big",
+      "instance_id": "0c7ec11f-977b-4027-a4d2-3dd81a9fbeb6"
+    }
+  }
+```
+
+The value of an open dependencies parameter is an array of objects, following
+the structures above. Additionally, the platform MAY add a dependency name,
+set by the user to each object.
+
+| Field | Type | Description |
+| --- | --- | --- |
+| name* | string | Dependency name. |
+
+\* Fields with an asterisk are REQUIRED.
+
+```
+{
+  "parameters": {
+    "some_dependencies ": [
+      {
+        "name": "master-database",
+        "service_id": "71ef3933-8520-4f26-b808-f483d23f94a6",
+        "service_name": "mysql",
+        "plan_id": "1b367d58-6f17-4cb3-9ef8-3a69e752b826",
+        "plan_name": "small",
+        "credentials": {
+          "uri": "mysql://mysqluser:pass@mysqlhost:3306/dbname",
+          "username": "mysqluser",
+          "password": "pass",
+          "host": "mysqlhost",
+          "port": 3306,
+          "database": "dbname"
+        }
+      },
+      {
+        "name": "internal-store",
+        "service_id": "90c7c453-c6a3-47ef-8284-5a2c00f4f18b",
+        "service_name": "the-database",
+        "plan_id": "d9301d66-7621-4526-9b97-63e93ddc2d13",
+        "plan_name": "big",
+        "instance_id": "0c7ec11f-977b-4027-a4d2-3dd81a9fbeb6"
+      }
+    ]
+  }
+}
+```
+
+### Dependencies
+
+| Field | Type | Description |
+| --- | --- | --- |
+| open\_dependencies\_parameter | string | A parameter name used for an open list of dependencies. |
+| services | [array-of-service-dependency-objects](#service-dependencies) | A list of service dependencies. |
+| instances | [array-of-instance-dependency-objects](#instance-dependencies) | A list of instance dependencies. |
+
+\* Fields with an asterisk are REQUIRED.
+
+### Service Dependencies
+
+| Field | Type | Description |
+| --- | --- | --- |
+| parameter* | string | Name of the parameter. MUST be a non-empty string and MUST be unique within all dependency parameters. |
+| description* | string | A short description of the dependency. MUST be a non-empty string. |
+| optional  | boolean | Whether the dependency is optional or not. Defaults to false. |
+| service_id | string | Service ID of the instance. MUST be a non-empty string. MUST NOT be used in conjunction with the fields `plan_ids` and `service_name`. |
+| plan_ids | array-of-strings | An array of plan IDs. The instance must match one of those plans. MUST be a non-empty array of non-empty strings. MUST NOT be used in conjunction with the fields `service_id` and `service_name`. |
+| service_name | string | Service name of the instance. MUST be a non-empty string. MUST NOT be used in conjunction with the fields `service_id` and `plan_ids`. |
```

Comment:
Should this be a list?

The spec says this:

>Additionally, some Platforms might modify the service names before presenting them to their users.

So the platform will have to also translate this name to have it make sense to the user.